.env.example

```diff
@@ -17,9 +17,6 @@ APP_FORCE_HTTPS=false
 # Do note that this disable CSP!!
 DEBUGBAR_ENABLED=false
 
-# enable or disable the v6 layout.
```

Comment:
hide this option, we do not want to advertise that anymore. Let's make the vue front-end strong default.